### PR TITLE
[#1] First cut on an alpha prototype

### PR DESCRIPTION
**Pull request by kardan**
- Added a simple REST API with Liberator
- Added basic message queue event handler
- Created a crude namespace data-store for fixture data
- Made sure the app can run from lein and as a jar
- Updated README with more content

### CODE REVIEW

**Comment by iperdomo** [comment]
As discussed in our conversation:
- Amend the copyright year
- Routes shouldn't contain a trailing slash
- Double check the usage of spaces vs tabs on `project.clj`

**Comment by kardan** [comment]
Updated "feature/1-setup" after review.

**Comment by kardan** [comment]
Honestly I don't think it is "pernickety", it's the services' interface (ok, maybe a bit - but it's still important ;-) ). I initially added slashes while I was investigating how strict Liberator was with them and forgot to remove them. So I'm in the non trailing slash religion, but got caught slashing...

I don't think it's a matter of being pedant with URLs, but more about preference. One could just as well argue that /services returns "a" resource that list the services. And that the slashes represent the hierarchy up to that resource. It's more common with APIs, but even blogs sometimes use of non trailing slashes on a [category list](http://roy.gbiv.com/untangled/tag/opensolaris).

File: README.md [hunk 1]
```diff
@@ -1,8 +1,45 @@
...
+To run Akvo-notifications issue
+```bash
+$ lein ring server
+´´´
+That should open a browser window with http://localhost:3000
+
+## REST API
+Available routes, you will need trailing slashes.
+
+/services/
```
Comment (iperdomo) [inline README.md:40]: the documentation still has the trailing slash

File: src/akvo/notifications/rest_api.clj [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+(ns akvo.notifications.rest-api
+  ^{:doc "REST API for other services to consume."}
+  (:require [liberator.core :refer [defresource resource]]
+            [compojure.core :refer [defroutes ANY]]
+            [compojure.route :as route]
+            [ring.middleware.params :refer [wrap-params]]
+            [akvo.notifications.data-store :as data]))
+
+(def available-media-types
```
Comment (iperdomo) [inline src/akvo/notifications/rest_api.clj:26]: are we going to support all this media types?

Comment (kardan) [inline src/akvo/notifications/rest_api.clj:26]: No, I only added a couple as an example. I guess future discussions about serialisation and architecture will steer what types we expose.

**Comment by iperdomo** [comment]
:+1: code reviewed + correct

**Pull request merged**